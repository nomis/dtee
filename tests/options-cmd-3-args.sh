#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

# Numbers are out of order to check they're not being sorted
run_test_once "--debug-options" "command" "argument 3" "argument 1" "argument 2"
RET=$?

echo RET $RET
if [ $RET -eq 0 ]; then
	exit 0
fi
exit 1
