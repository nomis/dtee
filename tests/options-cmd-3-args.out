-=command
-=argument 3
-=argument 1
-=argument 2
debug-options=1
